accounts/abi/abi.go

```diff
@@ -166,6 +166,9 @@ func (abi ABI) Call(executer Executer, v interface{}, name string, args ...inter
 }
 
 var interSlice = reflect.TypeOf([]interface{}{})
+var r_hash = reflect.TypeOf(common.Hash{})
+var r_bytes = reflect.TypeOf([]byte{})
+var r_byte = reflect.TypeOf(byte(0))
```

Comment:
Also possible call it `r_interfaces` to make it consistent with `byte` / `bytes`